### ISSUE TITLE
ask about error_record

### ISSUE THREAD

**Issue by thangckt** [MDIL-SNU/SevenNet#158]
hi @YutackPark 

I have a little confuse about `error_record`. My setting as below:
```
 loss: 'mse' 
 force_loss_weight: 1.0    
 stress_loss_weight: 1.0  
 error_record:
                - ['Energy', 'RMSE']
                - ['Force', 'RMSE']
                - ['Stress', 'RMSE']
                - ['TotalLoss', 'None']
                - ['Energy', 'Loss']
                - ['Force', 'Loss']
                - ['Stress', 'Loss']
```
then, get output as 

|trainset_Energy_RMSE | trainset_Force_RMSE | trainset_Stress_RMSE | trainset_TotalLoss | trainset_Energy_Loss | trainset_Force_Loss | trainset_Stress_Loss |
|-|-|-|-|-|-|-|
0.151689| 0.357032| 16.601852| 45.974178| 0.023007| 0.042483| 45.908688

I can get `trainset_Energy_Loss = trainset_Energy_RMSE^2`

but `trainset_Force_Loss != trainset_Force_RMSE ^2`

Can we expect `trainset_Force_Loss = trainset_Force_RMSE ^2`? Or they should not equal?
Can you have a little help?

Thank you so much.

**Comment by YutackPark** [comment]
The loss recorded here is the mean (about training steps) of the mean (about batch) while RMSE error is a true mean of all structures or atoms. Sevennet uses running statistics to compute the error. Therefore you will get the same Loss and RMSE if your data set equals batch size. I have also found this issue before, but I decided to keep it as there exists some ambiguity and challenge; 1) by definition, the loss is divided by batch size, and 2) for flexibility, the current implementation of loss error recording directly reads scalar outputs of loss, so it is not bothered by how loss is defined.

I'll document this when I have a chance.
